src/routes/docs/tutorials/android/step-2/+page.markdoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: tutorial
+title: Initialize project
+description: Learn to build an Android app with no backend code using an Appwrite backend.
+step: 2
+difficulty: beginner
+---
+
+# Initialize Android Studio project {% #initialize-android-studio-project %}
+
+Open Android Studio and click **New Project** to create a new project. Select **Empty Views Activity**, and click **Next**.
```

Comment:
Update screenshot, this is outdated version of AS

Reply:
![image](https://github.com/appwrite/website/assets/37345795/fc8029f0-f672-4695-a63d-7aed0a506586)

Here's the new screenshot. How am I supposed to make this change in this branch though?

Reply:
@letsintegreat Oh no you don't have to, I'll be making these changes :)